msgpack/ext.py

```diff
@@ -55,9 +48,9 @@ def __init__(self, seconds, nanoseconds=0):
 
         Note: Negative times (before the UNIX epoch) are represented as negative seconds + positive ns.
         """
-        if not isinstance(seconds, int_types):
+        if not isinstance(seconds, int):
             raise TypeError("seconds must be an interger")
```

Comment:
there is a typo.

Reply:
I don't see the typo. What is it?
In case you talk about the change `int_types` to `int`, `int_types` was a variable needed previously but its value is always `int` with python3 so `int_types` is removed by this PR.

Reply:
interger

Reply:
It's fixed.